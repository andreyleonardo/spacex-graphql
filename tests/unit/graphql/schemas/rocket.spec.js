const glue = require('schemaglue');
const { schema } = glue('src/graphql/rocket');

describe('Rocket Schema', () => {
  it('schema matches with snapshot', () => {
    expect(schema).toMatchSnapshot();
  });
});
