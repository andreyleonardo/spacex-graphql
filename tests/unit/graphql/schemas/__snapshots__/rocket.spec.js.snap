// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rocket Schema schema matches with snapshot 1`] = `
"type Rocket {
  rocket_id: String
  rocket_name: String
  rocket_type: String
}
type Query {

  rockets: [Rocket]
}"
`;
